src/components/subscriptions/licenses/LicenseManagementTable/LicenseManagementRemindModal.jsx

```diff
@@ -0,0 +1,199 @@
+      })
+      .catch((error) => {
+        logError(error);
+        setRequestState({ ...initialRequestState, error });
+      });
+  };
+
+  const handleClose = () => {
+    if (!requestState.loading) {
+      onClose();
+    }
+  };
+
+  const getRemindButtonState = () => {
+    if (requestState.error) {
+      return 'error';
+    }
+    if (requestState.loading) {
+      return 'pending';
+    }
+    return 'default';
+  };
+
+  return (
+    <ModalDialog
+      title={title}
+      isOpen={isOpen}
+      onClose={handleClose}
+      hasCloseButton={false}
+    >
+      <ModalDialog.Header>
+        <ModalDialog.Title>
+          {title}
+        </ModalDialog.Title>
+      </ModalDialog.Header>
+      <ModalDialog.Body>
+        {requestState.error
+            && (
+            <Alert variant="danger">
+              <p>There was an error with your request. Please try again.</p>
+              <p>
+                If the error persists,&nbsp;
+                <Hyperlink destination={configuration.ENTERPRISE_SUPPORT_URL}>
+                  contact customer support.
+                </Hyperlink>
+              </p>
+            </Alert>
+            )}
+        <h3>Email Template</h3>
+        <Form>
+          <Form.Group controlId="email-template-greeting">
+            <Form.Label>Customize Greeting</Form.Label>
+            <Form.Control
+              rows={3}
+              as="textarea"
+              data-hj-suppress
+              value={emailTemplate.greeting}
+              onChange={(e) => setEmailTemplate({ ...emailTemplate, greeting: e.target.value })}
+            />
+          </Form.Group>
+          <Form.Group controlId="email-template-body">
+            <Form.Label>Body</Form.Label>
+            <Form.Control
+              rows={3}
+              as="textarea"
+              data-hj-suppress
+              value={emailTemplate.body}
+              readOnly
+            />
+          </Form.Group>
+          <Form.Group controlId="email-template-closing">
+            <Form.Label>Customize Closing</Form.Label>
+            <Form.Control
+              rows={3}
+              as="textarea"
+              data-hj-suppress
+              value={emailTemplate.closing}
+              onChange={(e) => setEmailTemplate({ ...emailTemplate, closing: e.target.value })}
+            />
+          </Form.Group>
+        </Form>
+
+      </ModalDialog.Body>
+      <ModalDialog.Footer>
+        <ActionRow>
+          <ModalDialog.CloseButton variant="tertiary">
+            Cancel
+          </ModalDialog.CloseButton>
+          <StatefulButton
+            state={getRemindButtonState()}
+            variant="primary"
+            onClick={handleSubmit}
+            {...{
+              labels: {
+                default: `Remind (${numberToRemind})`,
+                pending: `Reminding (${numberToRemind})`,
+                complete: 'Done',
+                error: `Retry Remind (${numberToRemind})`,
```

Comment:
nit: "Retry remind" (lowercase "remind") since only the first letter of button CTAs should be capitalized.